interface/models.py

```diff
@@ -152,7 +154,7 @@ class Submission(models.Model):
     history = HistoricalRecords()
 
     def update_state(self):
-        if self.state == self.STATE_DONE or self.evaluator_job_id is None:
+        if self.state == self.STATE_DONE or self.evaluator_job_id is None or self.STATE_ERROR:
```

Comment:
What you can do here is the following.
``` if self.state in [self.STATE_DONE, self.STATE_ERROR] or ...```
Hmm...could you check if adding this makes it working?

Reply:
You nailed it, this solved the problem @gmuraru !

<img width="1217" alt="Screenshot 2020-10-04 at 21 18 30" src="https://user-images.githubusercontent.com/22433243/95030599-38e7d080-0687-11eb-90d7-11c1545dab10.png">

Didn't imagine a double `or` in a if statement could lead to this in Python 🤔